<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filteredTeams</class>
 <widget class="QDialog" name="filteredTeams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1618</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>1101</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;
color: rgb(0, 101, 101);

</string>
   </property>
  </widget>
  <widget class="QLabel" name="accumulatorLabel">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>391</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="noFilter_radioButton">
      <property name="text">
       <string>Display with no filter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="filterByAmericanLeague_radioButton">
      <property name="text">
       <string>Display American League</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="filterByNationalLeague_radioButton">
      <property name="text">
       <string>Display National League</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="filterByTypology_radioButton">
      <property name="text">
       <string>Display Stadium sorted by park typology</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="filterByOpenRoof_radioButton">
      <property name="text">
       <string>Display Stadiums with open roof types</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="filterByDateOpen_radioButton">
      <property name="text">
       <string>Display Stadiums by date opened</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="filterBySeatingCapacity_radioButton">
      <property name="text">
       <string>Display Stadiums sorted by seating capacity</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="filterByDistancetoCF_radioButton">
      <property name="text">
       <string>Display stadiums sorted by distance to center field</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="filterMaxMin">
      <property name="text">
       <string>Max/Min</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>151</width>
     <height>591</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:checked {
	background-color: #4398D8;
}
QPushButton:hover{
	background-color: #85C1E9
}
#frame { border: 2px solid black;
border-radius:20px;
background-color:rgba(0,0,0,100); }

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="display_menu">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/icons/feather/arrow-right.svg</normaloff>
      <normalon>:/icons/feather/arrow-down.svg</normalon>:/icons/feather/arrow-right.svg</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="team_widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>120</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QPushButton" name="one_team">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>One Team</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filterTeam">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>50</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Filter Team</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Trip_button">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Trip</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/icons/feather/arrow-right.svg</normaloff>
      <normalon>:/icons/feather/arrow-down.svg</normalon>:/icons/feather/arrow-right.svg</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>250</y>
      <width>131</width>
      <height>261</height>
     </rect>
    </property>
    <widget class="QPushButton" name="DFS">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>0</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>DFS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BFS">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>BFS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MST">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>80</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>MST</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startDodger">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>120</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>START -- Dodger</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startMarlin">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>160</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>START -- Marlins</string>
     </property>
    </widget>
    <widget class="QPushButton" name="custom">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>200</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>CUSTOM</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="mainpage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
text-align: left;
border: none;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/icons/feather/home.svg</normaloff>:/icons/feather/home.svg</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="back_menu">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align: left;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/feather/menu.svg</normaloff>:/icons/feather/menu.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>back_menu</zorder>
  <zorder>tableView</zorder>
  <zorder>accumulatorLabel</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>back_menu</sender>
   <signal>clicked()</signal>
   <receiver>frame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>display_menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>team_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Trip_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
